Cargo.toml

```diff
@@ -25,6 +25,7 @@ jsonrpsee = "0.16.2"
 log = "0.4.17"
 once_cell = { version = "1.17.1", default-features = false }
 proptest = "1.1"
+rand = "0.8.4"
```

Comment:
Hmm... curious what we are using `rand` for...

Reply:
To create Genesis nominations, but since I'm using the same seed, it's not random.